source/features/split-issue-pr-search-results.tsx

```diff
@@ -28,7 +28,11 @@ function updateLinkElement(link: HTMLAnchorElement, type: GitHubConversationType
 function init(): void {
 	cleanLinks();
 
-	const issueLink = select('a.menu-item[href*="&type=issues"]')!;
+	const issueLink = select('a.menu-item[href*="&type=issues"]');
+	if (!issueLink) {
+		return;
+	}
```

Comment:
When does this happen?

Reply:
I mentioned this in the commit message: https://github.com/refined-github/refined-github/search

Reply:
We should use a URL-based exclusion instead because this silences any future errors we might have. Example:

```js
eclude: [
	() => location.search === ''
]
```

No need for a full, custom detection in github-url-detection